README.md

```diff
@@ -86,56 +86,62 @@ Usage: cloudos job run [OPTIONS]
-  --git-tag TEXT               The tag to run for the selected pipeline. If
-                               not specified it defaults to the last commit of
-                               the default branch.
-  --job-name TEXT              The name of the job. Default=new_job.
-  --resumable                  Whether to make the job able to be resumed or
-                               not.
-  --batch                      Whether to make use the batch executor instead
-                               of the default ignite.
-  --instance-type TEXT         The type of AMI to use. Default=c5.xlarge.
-  --instance-disk INTEGER      The amount of disk storage to configure.
-                               Default=500.
-  --spot                       Whether to make a spot instance.
-  --storage-mode TEXT          Either 'lustre' or 'regular'. Indicates if the
-                               user wants to select regular or lustre storage.
-                               Default=regular.
-  --lustre-size INTEGER        The lustre storage to be used when --storage-
-                               mode=lustre, in GB. It should be 1200 or a
-                               multiple of it. Default=1200.
-  --wait-completion            Whether to wait to job completion and report
-                               final job status.
-  --wait-time INTEGER          Max time to wait (in seconds) to job
-                               completion. Default=3600.
-  --wdl-mainfile TEXT          For WDL workflows, which mainFile (.wdl) is
-                               configured to use.
-  --wdl-importsfile TEXT       For WDL workflows, which importsFile (.zip) is
-                               configured to use.
-  -t, --cromwell-token TEXT    Specific Cromwell server authentication token.
-                               Only required for WDL jobs.
-  --repository-platform TEXT   Name of the repository platform of the
-                               workflow. Default=github.
-  --cost-limit FLOAT           Add a cost limit to your job. Default=-1 (no
-                               cost limit).
-  --verbose                    Whether to print information messages or not.
-  --help                       Show this message and exit.
+  -k, --apikey TEXT           Your CloudOS API key  [required]
+  -c, --cloudos-url TEXT      The CloudOS url you are trying to access to.
+                              Default=https://cloudos.lifebit.ai.
+  --workspace-id TEXT         The specific CloudOS workspace id.  [required]
+  --project-name TEXT         The name of a CloudOS project.  [required]
+  --workflow-name TEXT        The name of a CloudOS workflow or pipeline.
+                              [required]
+  --job-config TEXT           A config file similar to a nextflow.config file,
+                              but only with the parameters to use with your
+                              job.
+  -p, --parameter TEXT        A single parameter to pass to the job call. It
```

Comment:
@dapineyro a minimally impactful note on the code quality, but possibly substantially important for the PR review(er).

You introduced a new param injected in the help message versus appending in the end and this create a bigger git diff than what actually changed.

It would be ideal to stack new params at the end so that git only pick ups that difference vs the reordering.

Reply:
Yes @cgpu that makes sense. I found it somehow more logical to have it close to the previous --nextflow-profile command, but I see it creates git diff unnecessary issues. I'll take it into account for future new parameters :+1: